src/lib/yaraparse.py

```diff
@@ -136,15 +143,15 @@ def ursify_hex(hex_str: str) -> UrsaExpression:
             output.append(part[last_end:])
 
     core = "} & {".join(output)
-    return UrsaExpression(f"{{{core}}}")
+    return UrsaExpression.literal(core)
```

Comment:
It'll work, but it's a bit against the spirit of the idea (`core` is something like `616161} & {626262} & {636363`). Ideally I'd like to get rid of the default constructor. Maybe something like:

```
return UrsaExpression.and_(UrsaExpression.literal(f) for f in output)
```

Reply:
Done